src/components/FileUpload/index.tsx

```diff
@@ -82,6 +80,11 @@ const FileUpload = (props: FileUploadProps) => {
             <span className="usa-file-input__choose">Change file</span>
           </div>
         )}
+        {file && (
+          <div role="alert" aria-live="assertive" className="sr-only">
```

Comment:
Does this need `tabIndex={-1}`?

Not sure exactly how (or if) it would change. Asking because the diff below has it.

Reply:
I don't think we need that since the div wouldn't be focusable with tab normally. I'm not sure why we have it below TBH.